core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -638,8 +651,8 @@ public String toString() {
         .add("fileBlockInfos", mFileBlockInfos)
         .add("mountId", mMountId).add("inAlluxioPercentage", mInAlluxioPercentage)
         .add("ufsFingerprint", mUfsFingerprint)
-        .add("aclEntries", mAclEntries)
-        .add("defaultAclEntries", mDefaultAclEntries)
+        .add("acl", mAcl.toString())
```

Comment:
what if this is `null`?

Reply:
made the decision to not allow them to be `null`